code/modules/antagonists/_common/antag_team.dm

```diff
@@ -168,7 +168,8 @@ GLOBAL_LIST_EMPTY(antagonist_teams)
 		return
 
 	for(var/datum/mind/M as anything in members)
-		to_chat(M.current, "<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>")
+		SEND_SOUND(M.current, sound('sound/effects/adminhelp.ogg', 0, 1, 0, 25))
+		to_chat(M.current, chat_box_red("<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span><span class='notice'>[message]</span>"))
```

Comment:
```suggestion
		to_chat(M.current, chat_box_red("<font color='#d6000b'><span class='bold'>Admin Team Message ([user.key]): </span></font><span class='notice'>[message]</span>"))
```
I know that the second span class for the message will set a font color separate from the font tag, but font should still be closed as it is a container tag.